csharp/src/Drivers/Apache/Hive2/HiveServer2Statement.cs

```diff
@@ -396,9 +397,9 @@ protected virtual async Task<QueryResult> GetCrossReferenceAsForeignTableAsync(C
                 null,
                 null,
                 null,
-                EscapeUnderscoreInName(CatalogName),
-                EscapeUnderscoreInName(SchemaName),
-                EscapeUnderscoreInName(TableName),
+                (CatalogName),
+                (SchemaName),
+                (TableName),
```

Comment:
Was this supposed to be changed to `EscapePatternWildcardsInName` or does this Thrift API not work with wild cards? If the latter, then please remove the parens from the arguments.